src/components/v5/common/ActionsContent/partials/SocialLinksTable/types.ts

```diff
@@ -0,0 +1,29 @@
+import { UseFieldArrayReturn, FieldValues } from 'react-hook-form';
+
+export const LINK_TYPE = {
+  Custom: 'Custom',
+  Whitepaper: 'Whitepaper',
+  Youtube: 'Youtube',
+  Discord: 'Discord',
+  Twitter: 'Twitter',
+  Telegram: 'Telegram',
+  Github: 'Github',
+  Facebook: 'Facebook',
+  Instagram: 'Instagram',
+} as const;
+
+export type LinkType = (typeof LINK_TYPE)[keyof typeof LINK_TYPE];
+
+export interface SocialLinksTableProps {
+  name: string;
+}
+
+export interface SocialLinksTableModel {
+  key: string;
+}
+
+export type RowFieldProps = SocialLinksTableProps;
```

Comment:
This type shouldn't be added here. It should be add in the same directory as the component where it's used